src/org/labkey/test/components/ui/grids/ResponsiveGrid.java

```diff
@@ -572,7 +598,8 @@ protected final WebElement getColumnHeaderCell(String headerText)
         {
             if (!headerCells.containsKey(headerText))
             {
-                WebElement headerCell = Locator.xpath("//th[./span[contains(text(), '" + headerText + "')]]").findElement(this);
+                WebElement headerCell = Locator.tagWithClass("th", "grid-header-cell")
+                        .withChild(Locator.tag("span").startsWith(headerText)).findElement(this);
```

Comment:
Wouldn't `Locator.tag("span").withText(headerText))` be a better option? This would remove any ambiguity as to which column to get.

Reply:
I suspect that `withText `will likely fail if the cell has a menu with text inside of it?  I did try using `withText `before suggesting `startsWith`, but I didn't thoroughly investigate why it didn't work.  I'll give this some attention